superbench/benchmarks/micro_benchmarks/memory_copy_bw_cuda_performance.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Shall we change the file name to `cuda_memory_bw_performance.py`?

Reply:
changed